File: kiki616/ABC/A/309.py
A,B = map(int, input().split())

if B - A == 1:
    if A % 3 == 0:
        print("No")
    else:
        print("Yes")
else:
    print("No")

# if A + 1 == B or  A - 1 == B or  A + 3 == B or  A - 3 == B:
#     if A % 3 == 0 
    
#     or B % 3 == 0 or A % 3 == 1 or A % 3 == 1:
#         print("No")
#     elif :
#         print("Yes")
# else:
#     print("No")

File: OCH_Tao/ABC/A/309.js
const [A, B] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ").map(x => parseInt(x, 10));
if (A % 3 !== 0 && A + 1 === B) {
  console.log("Yes");
} else {
  console.log("No");
}